packages/webapp/src/webview/ui/components/DialogBoxes/MessageDialogBox/MessageDialogBox.scss

```diff
@@ -31,7 +31,7 @@
 
     .default-button {
```

Comment:
I'm wondering why we even need the class `.default-button` and `.primary-button`. This should all be in the standard implementation of the buttons from `@sap-ux/ui-components`, e.g.:
https://github.com/SAP/open-ux-tools/blob/ab66e8738c54dcd1cc9edb5a185cf24058fda9f0/packages/ui-components/src/components/UIButton/UIDefaultButton.tsx#L52

Perhaps this is a leftover from the time the components have been locally defined. I'd remove the classes completely and see if it is correct then.